packages/transaction-controller/src/utils.ts

```diff
@@ -43,6 +43,10 @@ export function normalizeTransaction(transaction: Transaction) {
       normalizedTransaction[key] = NORMALIZERS[key](transaction[key]) as never;
     }
   }
+
+  if (transaction?.type === '0x0') {
```

Comment:
Rather than hard coding this, should we not add an additional entry to `NORMALIZERS` above?

Reply:
Great feedback. Implemented [here](https://github.com/MetaMask/core/pull/1672). Will be closing this PR